src/main/features/response/views/confirmation/full-admission-steps.njk

```diff
@@ -0,0 +1,28 @@
+{% set claimantName = claim.claimData.claimant.name %}
+{% set claimantDetailsPageURI = ClaimPaths.claimantDetailsPage.evaluateUri({ externalId: claim.externalId }) %}
+
+{% if claim.response.paymentOption === domain.PaymentOption.IMMEDIATELY %}
+  <p>{{ t('When {{ claimantName }} gets your payment they must tell you and the court you’ve paid.', { claimantName: claimantName }) }}</p>
+{% endif %}
+
+{% if claim.response.paymentOption === domain.PaymentOption.FULL_BY_SPECIFIED_DATE %}
+  <p>{{ t('If {{ claimantName }} accepts your offer to pay, you need to <a href="{{ link }}">contact them for payment details</a> and to arrange payment.', { claimantName: claimantName, link: claimantDetailsPageURI }) | safe }}</p>
```

Comment:
should we be using internalLink macro for all these links?

Reply:
I investigated use of `internalLink` and found that it has major flaw in terms of using it with i18n framework.

First issue is that because of `<a href="{{ href }}">{{ text }}</a>` URI gets interpolated at the macro level which means that `<a href="/some-uri">text</a>` is being passed to i18n framework. That forces us to use hardcoded links in *.po files which means more maintenance.

Second issue is that HTML printed by the macro has line break at the and which means that without trimming entries in *.po files would need to have the same line break :(

```
{% macro internalLink(text, href) %}
  <a href="{{ href }}">{{ text }}</a><-- here
{% endmacro %}
```

The same flaw occurs in `externalLink` macro :(

Due to the above I won't be using link macros in the PR.